platform/commonUI/general/src/controllers/DateTimeFieldController.js

```diff
@@ -96,7 +96,14 @@ define(
                 updateFromView($scope.textValue);
             }
 
+            function forceBlur($event) {
+                if ($event.keyCode === 13) {
```

Comment:
Just a nitpick, wouldn't it be better to extract to a variable name that indicates what value `13` is besides a unlucky number? 🏷